File: Iniciante/URI 1008.py
salario = None
funcionarios = None
valorPorHora = None
horas = None

def read_integer():
  try:
    # read for Python 2.x
    return int(raw_input())
  except NameError:
    # read for Python 3.x
    return int(input())

def read_numeric():
  try:
    # read for Python 2.x
    return float(raw_input())
  except NameError:
    # read for Python 3.x
    return float(input())

funcionarios = read_integer()
horas = read_integer()
valorPorHora = read_numeric()
salario = horas * valorPorHora
print(str("NUMBER = ") + str(funcionarios))
print(str("SALARY = U$ ") + str("{:0.2f}".format(salario)))

File: Iniciante/URI 1008.hs
import Text.Printf

main :: IO ()
main = do
    funcionarios' <- getLine
    horas' <- getLine
    valorPorHora' <- getLine

    let funcionarios = read funcionarios' :: Int
    let horas = read horas' :: Double
    let valorPorHora = read valorPorHora' :: Double
    let salario = horas * valorPorHora

    putStrLn $ "NUMBER = " ++ show funcionarios
    putStrLn $ printf "SALARY = U$ %.2f" salario